python/deleteProject.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+# Generic/Built-in
+
+# Other Libs
+
+
+
+# Owned
+from capellaAPI.CapellaAPICommon import MyParser
+from capellaAPI.CapellaAPICommon import capella_logging
+from capellaAPI.CapellaAPICommon import check_if_valid_uuid
+from capellaAPI.CapellaAPI import CapellaAPI
+
+
+__author__ = 'Jonathan Giffard'
+__copyright__ = 'Copyright 2021, Couchbase'
+__credits__ = ['Jonathan Giffard']
+__license__ = 'MIT License'
+__version__ = '0.3.0'
+__maintainer__ = 'Jonathan Giffard'
+__email__ = 'jonathan.giffard@couchbase.com'
+__status__ = 'Dev'
+
+
+
+def main(CmdLineArgs):
+    cappella_api = CapellaAPI()
+
+    if CmdLineArgs.debug:
```

Comment:
You probably aren't going to change this now, but you could have made log level an env var given that all the commands can accept it.

Reply:
true , that's something I didn't consider.   I did want the flexibility of making it easy to toggle debug on / off via the command line, primarily so people who use the code samples can see what calls are being made and the response back.